code/modules/organs/blood.dm

```diff
@@ -265,7 +265,16 @@ var/const/BLOOD_VOLUME_SURVIVE = 122
 
 	if (iscultist(src))
 		B.data["occult"] = mind
-
+	
+	//take sample of changeling blood if mob is changeling
+	if (mind.GetRole(CHANGELING))
+		var/datum/reagent/C = new /datum/reagent/blood/changeling_blood
```

Comment:
Why would you do it like this instead of just having it make changeling blood from the start?

Reply:
Side note: I notice Dr. Acula (bloodbot) does not use takeblood, so it won't work with this. This isn't your fault but you do need to fix it.